src/coreclr/jit/emitxarch.cpp

```diff
@@ -10722,6 +10722,20 @@ void emitter::emitDispInsHex(instrDesc* id, BYTE* code, size_t sz)
     }
 }
 
+// emitDispEmbBroadcastCount: Display the tag where embedded broadcast is activated to show how many elements are broadcasted.
+//
+// Arguments:
+//   id - The instruction descriptor
+//
+void emitter::emitDispEmbBroadcastCount(instrDesc* id)
+{
+    ssize_t baseSize = GetInputSizeInBytes(id);
+    id->idClearEvexbContext();
+    ssize_t vectorSize = (ssize_t)emitGetMemOpSize(id);
+    id->idSetEvexbContext();
+    printf(" {1to%d}", vectorSize / baseSize);
```

Comment:
Can we make this a comment instead, so that it stays mostly valid assembly?

Something like `; broadcast x%d` instead, so this would be `vaddps   zmm0, zmm0, dword ptr [reloc @RWD00] ; broadcast x16`

Reply:
Isn't the `{1toX}` notation itself valid assembly?

Reply:
Ah, so it is. I've seen compilers not include that and instead infer it from the address size (e.g. `dword ptr [addr]` vs `zmmword ptr [addr]`).